content/neuvector-prime/azure/contents.lr

```diff
@@ -104,6 +104,55 @@ The Neuvector login will appear, login with the credential which you have set du
 
 Once logged in, you can begin to [navigate and configure NeuVector](https://open-docs.neuvector.com/navigation/navigation).
 
+
+
+```
+Important! The current Neuvector deployment exposes the original password in the Cluster 
+           configuration settings. Until this is resolved, it is suggested to change the
+           Admin password after initial login. Edit profile in the Neuvector Dashboard
```

Comment:
Do you think it might help if we add a bit more detail?  Something like:

1) Login to the NV dashboard
2) Click on Edit Profile, Type in the current password and new passwords
3) Click ......

Reply:
Included the following steps..

1. Access Neuvector Console
2. Login as username: admin password: <password which was set during the deployment>
3. Click Admin user dropdown and click My Profile
4. Click Edit Profile and set New password and Update Profile.

Reply:
LGTM